FEAT (#11) : update the /work-with-us page

activate-glacier-instinct opened the pull request:
Temporary PR for the /work-with-us work

activate-glacier-instinct committed:
FEAT (ChakraNextImage) : adds a new comp for handling NextJS image components

activate-glacier-instinct committed:
FEAT (ConsultImage) : adds a new Image type just for the Consult page

Also added borderRadius to ChakraNextImage

activate-glacier-instinct committed:
STYLES (consult/intro) : tweaked the spacing on the introduction

Also update the image styles to match closer to the Figma file

activate-glacier-instinct committed:
FIX : adds gitignore line for VSCode files

activate-glacier-instinct committed:
FEAT (consult): implements the new image style

activate-glacier-instinct committed:
STYLES: (consult) fixes the text styles on consult

activate-glacier-instinct committed:
FEAT (work-with-us/CTA) : adds a new CTA component

CTA component is simply a copy of the form on each card

Broke this up into a separate component so that I can easily style it up

Implemented this new component across all the new sections

activate-glacier-instinct committed:
FEAT (CTA) : updates the CTA to be a link to each guild

Updates all the guild links

activate-glacier-instinct committed:
FIX : updates the order of the tabs on a few sections

activate-glacier-instinct committed:
FEAT (work-with-us): updates the section images

Adds new assets

activate-glacier-instinct commented:
![localhost_3000_work-with-us (2)](https://user-images.githubusercontent.com/101533082/166126079-01689839-1622-4734-b126-58ecdc9e261a.png)

activate-glacier-instinct commented:
Done:
- design tweaks on images, text, spacing
- all images uploaded
- all first links added

Todo:
- [x] Carousels are not working atm, cannot change slides to a new on eg. click on Academy the slide doesn't update
- [x] Update content for each service, once the carousel is working
- [x] Fix any spacing and padding to match designs

activate-glacier-instinct committed:
FIX (CTA) : updates the html structure to make the CTA properly clickable

ken333135 reviewed (commented):
Looks great to me!

CompositeFellow commented:
I second this one

activate-glacier-instinct committed:
REFACTOR (CTA) : cleaned up the CTA component

activate-glacier-instinct committed:
FEAT (work-with-us) : adds carousels data

Each carousel content for the whole page

Data includes each slide data per carousel

activate-glacier-instinct committed:
FEAT (Carousel) : adds new carousel component

Makes the whole page more dymanic by taking data and converting it into an alternating list of carousel components

activate-glacier-instinct committed:
REFACTOR (work-with-us) : implements the new Carousel component

activate-glacier-instinct committed:
REFACTOR (Carousel) : split up the Carousel into two components Carousel and CarouselSlide

Laying down the foundation to start working on more complex CarouselSlide features

activate-glacier-instinct committed:
FEAT (Carousel) : adds the offset styling depending on carousel index

Decided against actually switching the Carousel slides order too - it seemed too confusing

activate-glacier-instinct committed:
FIX (work-with-us) : adds academy content

Also fixes the card alignment

activate-glacier-instinct committed:
CONTENT (work-with-us) : adds content for the bounty-board

activate-glacier-instinct committed:
CONTENT (work-with-us) : adds DAO Dash content

activate-glacier-instinct committed:
CONTENT (work-with-us) : adds content for the writers guild

activate-glacier-instinct committed:
CONTENT : adds State of the DAOs content

activate-glacier-instinct committed:
CONTENT (work-with-us) : adds content for the design guild

activate-glacier-instinct committed:
CONTENT (work-with-us) : adds content for the Podcast sponsorships

activate-glacier-instinct committed:
CONTENT (work-with-us) : adds content for consulting

activate-glacier-instinct committed:
CONTENT (work-with-us) : adds content for global tax

activate-glacier-instinct committed:
CONTENT (work-with-us) : tax image update

activate-glacier-instinct committed:
FIX (work-with-us) : fixed layout per page

activate-glacier-instinct committed:
REFACTOR : removes redundant code

activate-glacier-instinct committed:
FIX (work-with-us) : fixes the responsive layout

activate-glacier-instinct committed:
STYLES (work-with-us) : improves intro and general work-with-us>

Updates the layout to work across desktop, tablet, mobile

Add new font packages for Clear Sans

Updates the introduction spacing and text responsive rules

activate-glacier-instinct committed:
STYLES (work-with-us) : updates carousel styles for responsive behaviour

activate-glacier-instinct committed:
CONTENT (work-with-us): adds optimised image content

activate-glacier-instinct commented:
Ready for a proper review now. 

Test URL: https://bankless-website-7jtdojlqm-activate-glacier-instinct.vercel.app/work-with-us

All above tasks complete:
- [x] carousels
- [x] content (as much as could scrounge)
- [x[ responsiveness
- [x] first pass image optimisations

Having trouble with all the content loading during Google Dev Tool screen capture - so images are missing. Best to checkout the deployed version above.

Screenshots:
![desktop](https://user-images.githubusercontent.com/101533082/166391223-0c09b791-ed16-4431-96a8-f054d8fb68ab.png)
![tablet](https://user-images.githubusercontent.com/101533082/166391227-ca6cae89-9d39-454e-9b49-fb1ed51445f4.png)
![mobile](https://user-images.githubusercontent.com/101533082/166391231-5e89f9b6-779b-4305-b687-f4a58396cc49.png)

CompositeFellow commented:
minor change.  Can we update the bounty board link to go to https://www.bountyboard.xyz/ instead of https://bountyboard.bankless.community/ .  The community site will soon be obsolete

activate-glacier-instinct committed:
FIX (PageContent + page templates) : included the responsive styles in the PageContent component

Implemented the PageContent within _app

Removed all previous instances of the PageComponent on each individual page

activate-glacier-instinct committed:
CONTENT (work-with-us) : updates Custom Designs and Global Tax consultancy

Also add a new link for the bounty-board

ghost commented:
Looks good to me

activate-glacier-instinct commented on assets/content/pages/work-with-us/carouselContent.json:
```diff
@@ -0,0 +1,204 @@
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-12.jpg",
+          "alt": "Photo by Jason Goodman on Unsplash"
```
Not fully happy with this but yeah worth attributing where possible.

I'm thinking of changing it to a <figure> where the <figcaption> is the attribution

That way we can rewrite the alt as an actual alt :shrug:
